validation/provisioning/rke2/defaults/defaults.yaml

```diff
@@ -0,0 +1,81 @@
+rancher:
```

Comment:
If not already added, might be good to add a section in the READMEs explaining what these are.